docs/tutorials/isolation_forest_anomaly_detection.md

```diff
@@ -1,74 +1,66 @@
 # Monitor anomalies with a custom Isolation Forest metric
 
-On this page you will learn how to create a custom anomaly detection metric for a specific use case. 
-
-@@toc { depth=1 }
+On this page you will learn how to create a custom anomaly detection metric for a specific use case.
 
 ## Overview
 
 For this use case we have chosen a sample regression problem. We will monitor the model, which will predict how many taxi pickups will occur in the next hour, based on observations from past 5 hours. As a data source we will use a [dataset](https://www1.nyc.gov/site/tlc/about/tlc-trip-record-data.page) from NYC Taxi & Limousine Commission.
 
 ## Before you start
 
-We assume you already have a @ref[deployed](../../install/platform.md) instance of the Hydrosphere platform and a @ref[CLI](../../install/client/cli.md) on your local machine.
+We assume you already have a @ref[deployed](../installation/) instance of the Hydrosphere platform and a @ref[CLI](../installation/cli.md) on your local machine.
 
-To let `hs` know where the Hydrosphere platform runs, configure a new `cluster` entity. 
+To let `hs` know where the Hydrosphere platform runs, configure a new `cluster` entity.
 
-```sh 
+```bash
 $ hs cluster add --name local --server http://localhost
 $ hs cluster use local
 ```
 
-Also, you should have a target regression model already deployed on the cluster. You can find the source code of the target model here. 
-
+Also, you should have a target regression model already deployed on the cluster. You can find the source code of the target model here.
 
 ## Model training
 
-As a monitoring model we will use an autoregressive stateful IsolationForest model, which will be continuously retrained on a window of 5 consequent data samples. 
+As a monitoring model we will use an autoregressive stateful IsolationForest model, which will be continuously retrained on a window of 5 consequent data samples.
 
-We will skip most of the data preparation steps, for the sake of simplicity. 
+We will skip most of the data preparation steps, for the sake of simplicity.
 
-Python
-:   @@snip[train.py](snippets/isolation_forest_anomaly_detection/train.py) { #main-section }
+Python : @@snip[train.py](https://github.com/Hydrospheredata/hydro-serving/tree/54b7457851ad9de078cd092f083b8492dea6edca/docs/tutorials/monitoring/snippets/isolation_forest_anomaly_detection/train.py) { \#main-section }
```

Comment:
The best way - set tag for this commit and set link by ref?tag

Reply:
Same here, this is an old file removed from the documentation index.